examples/todoapp/fastlane/Fastfile

```diff
@@ -0,0 +1,569 @@
+require 'yaml'
+
+# Apple Team ID associated with your Apple account
+# https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/
+APPLE_DEVELOPMENT_TEAM = 'ZNE622G954'
+
+# The ID of the private App Store Connect Api Key
+# https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api
+IOS_P8_AUTH_KEY_ID = 'BX5CP94DHB'
+IOS_P8_AUTH_KEY_ISSUER_ID = '69a6de93-4129-47e3-e053-5b8c7c11a4d1'
+
+# The path of the directory in which the project is located
+PROJECT_DIRECTORY_PATH='.'
+
+# The url of the distribution repository (without http(s) prefix)
+DISTRIBUTION_REPOSITORY_URL='github.com/JvnSlv/todo-app-credentials.git'
```

Comment:
Please use the https://github.com/Prime-Holding/test-distribution-credentials repository

Reply:
Done